frontend/src/store/socket/helper.js

```diff
@@ -118,7 +118,11 @@ export function createSocket (state, context) {
     switch (reason) {
       case 'io server disconnect': {
         logger.debug('socket was forcefully disconnected by the server')
-        reconnect()
+        if (authnStore.isExpired()) {
+          authnStore.signout()
```

Comment:
for me it's not clear why we do the signout only in case when `io server disconnect` and why not in any case if it is expired

Reply:
The important thing here is that we do not try to reconnect if the session is expired. The `io server disconnect` is the case when the API token needs to be refreshed. This should happen if the absolute lifetime of the session is over. We counld also remove the signout here or move it outside of the case. But the other caes will not occur I assume.

Reply:
moved the signout at the bottom of the handler.